services/application/src/actions/user/send-login-link.js

```diff
@@ -54,57 +56,39 @@ module.exports = async ({
   const addressFields = ['name', 'streetAddress', 'city', 'regionName', 'postalCode'];
   const addressValues = addressFields.map(field => company[field]).filter(v => v).map(stripLines);
   const supportEmail = context.email || app.email || company.supportEmail;
+  const language = context.language || app.language || 'en-us';
   if (supportEmail) addressValues.push(supportEmail);
 
   const { token } = await createLoginToken({ applicationId, email: user.email, data: { source } });
   let url = `${authUrl}?token=${token}`;
   if (redirectTo) url = `${url}&redirectTo=${encodeURIComponent(redirectTo)}`;
 
-  const supportEmailHtml = supportEmail ? ` or <a href="mailto:${supportEmail}">contact our support staff</a>` : '';
-  const supportEmailText = supportEmail ? ` or contact our support staff at ${supportEmail}` : '';
+  const templateByLanguage = {
+    'en-us': englishTemplate({
+      sendingDomain: SENDING_DOMAIN,
+      supportEmail,
+      url,
+      appName,
+      addressValues,
+    }),
+    'es-mx': spanishTemplate({
+      sendingDomain: SENDING_DOMAIN,
+      supportEmail,
+      url,
+      appName,
+      addressValues,
+    }),
+  };
+
+  const templateForLanguage = templateByLanguage[language];
+  if (!templateForLanguage) throw createError(404, `No template available for language: ${language}!`);
+  const { subject, html, text } = templateForLanguage;
 
```

Comment:
Lets clean this part up a bit -- create an `email-templates/login-link/index.js` file that imports and exports the language functions.

Then this whole section can be condensed to something like this:
```js
if (!template[language]) throw createError(404, ...);
const { subject, html, text } = template[language]({ sendingDomain, ... });
```